packages/component-base/test/polylit-mixin.test.js

```diff
@@ -531,9 +526,13 @@ describe('PolylitMixin', () => {
           `;
         }
 
-        _valueOrLoadingChanged(_value, _loading) {}
+        _valueOrLoadingChanged(_value, _loading) {
+          valueOrLoadingChangedSpy(_value, _loading);
```

Comment:
nit: no need to use `_` as it was previously used to suppress ESLint warnings.

```suggestion
        _valueOrLoadingChanged(value, loading) {
          valueOrLoadingChangedSpy(value, loading);
```